blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md

```diff
@@ -0,0 +1,772 @@
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/internal-fn.def" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/ansidecl.h" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/tree-scalar-evolution.h" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/gimple-predict.h" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./libexec/gcc/aarch64-buildroot-linux-uclibc/7.4.0/install-tools/mkheaders" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/langhooks.h" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/lra.h" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/dbxout.h" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./bin/aarch64-buildroot-linux-uclibc-gcc.br_real" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/profile.h" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/highlev-plugin-common.h" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/tree-hasher.h" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  Warning: host file "./lib/gcc/aarch64-buildroot-linux-uclibc/7.4.0/plugin/include/cfg-flags.def" is touched by more than one package: [u'host-gcc-initial', u'host-gcc-final']
+  ```
+
+4. Now our image has to be flashed at SD card.
+
+    ```bash
+    sudo dd if=output/images/sdcard.img of=/dev/sdX bs=4M conv=fsync
+    ```
+
+     > `/dev/sdX` means a USB adapter device with SD card for example `sdc`
+
+5. Check how it looks like at a HummingBoard
+
+    ```bash
+    sudo minicom -b 115200 -D /dev/ttyUSBx
+    ```
+
+    > Note: you need to connect a MicroUSB port with PC
+
+## Build and run a Yocto image
+SolidRun does not provide Yocto support for **imx8**.
+For this purpose repository `3mdeb/meta-imx8` was created. It contains
+BSP layers for **HummingBoard Pulse** now only, but support for
+other **imx8** platforms will be available in the future.
+
```

Comment:
We might also push support for this board to one of the already existing Yocto meta-layers.